imagecraft/pack/grubutil.py

```diff
@@ -155,5 +155,9 @@ def setup_grub(image: Image, workdir: Path, arch: str) -> None:
             grub_target=_ARCH_TO_GRUB_EFI_TARGET[arch],
             loop_dev=loop_dev,
         )
+    except errors.ChrootMountError as err:
+        # Ignore mounting errors indicating the rootfs does not have
+        # the needed structure to install grub.
+        emit.message(f"Cannot install GRUB on this rootfs: {err}")
```

Comment:
pedantic: these probably should be `emit.progress(..., permanent=True)` since they're warnings and not the final output (https://github.com/canonical/craft-cli/issues/248)

Reply:
Oh nice catch, thanks!